packages/blade/src/utils/useUpdateEffect.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const useUpdateEffect: typeof React.useEffect = (effect, deps) => {
```

Comment:
Add some comments here on what this does

Reply:
Yep just added! 👍